Source/DafnyDriver/Commands/StabilityCommand.cs

```diff
@@ -0,0 +1,94 @@
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Linq;
+using Microsoft.Boogie;
+
+namespace Microsoft.Dafny;
+
+public class StabilityCommand : ICommandSpec {
+  public IEnumerable<IOptionSpec> Options => new IOptionSpec[] {
+    IterationsOption.Instance,
+    RandomSeedOption.Instance,
+    FormatOption.Instance,
+    IsolateAssertionsOption.Instance,
+  }.Concat(CommandRegistry.CommonOptions);
+
+  public Command Create() {
+    var result = new Command("check-stability", "(experimental) Check the stability of the program proofs. Be aware that the name, options and output of this command will change in the future.");
```

Comment:
I wonder whether it would make sense in the new CLI framework to allow certain commands or options to be programmatically marked as experimental. This would make it possible to disallow experimental features in settings where change is particularly disruptive.